patches/eslint-plugin-react-hooks+4.0.0.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.development.js b/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.development.js
```

Comment:
@swissspidy is there no way at all here to actually put the pull request reference? Or even maybe automatically pull the diff from the pull request? I'm asking since I just rebased the pull request and I'll have to do it probably again and again for some time until it's accepted, hopefully.

Reply:
I don't think there's any automated way for this, but I'll have a look.